docs/index.rst

```diff
@@ -25,6 +27,18 @@ The *fledge-south-dnp3* allows Fledge to act as a DNP3 master and gather data fr
 
   - **Network timeout**: Timeout for fetching data from the Out Station expressed in seconds.
 
+  - **DNP3 debug objects**: DNP3 communication and data objects logging. Detailed informations require plugin log level set to 'info' or 'debug'.
+
++----------+
+| |dnp3_2| |
++----------+
+
+  - **Outstations tab**: The outstations list for multiple connection. If an entry is added then the current outstation settings 'Outstation addres', 'Outstation port' and 'Outstation link Id' are ignored.
```

Comment:
A list of outstations. Each item in the list consists of three properties, the Outstation address, port and link id. If this list is populated with one or more outstations then the global 'Outstation address', 'Outstation port' and 'Outstation link Id' will be ignored.